packages/x64dbg.dbgchild.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,61 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+try {
+    $toolSrcDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+    $toolDir = Join-Path ${Env:RAW_TOOLS_DIR} 'x64dbg\release' -Resolve
+    $packageArgs = @{
+        packageName   = ${Env:ChocolateyPackageName}
+        unzipLocation = $toolSrcDir
+        url           = 'https://github.com/therealdreg/DbgChild/releases/download/beta10/DbgChild.Beta.10.zip'
+        checksum      = 'f17f588795d8f5f94d71335a8acfa58946bb03a94a5637be7f3e804c652ea2b4'
+        checksumType  = 'sha256'
+    }
+
+    VM-Remove-PreviousZipPackage ${Env:chocolateyPackageFolder}
+    Install-ChocolateyZipPackage @packageArgs
+    $toolSrcDir = Join-Path $toolSrcDir 'DbgChild Beta 10'
```

Comment:
```suggestion
    $unzippedDir = (Get-ChildItem -Directory $tempDownloadDir | Where-Object {$_.PSIsContainer} | Select-Object -f 1).FullName
```
[niptick] I think I would use a different variable name for clarity and also make it non dependent on the version so that this package gets automatically updated

Reply:
done!